packages/wallet-sdk/jest.config.ts

```diff
@@ -11,20 +11,12 @@ export default {
   collectCoverage: true,
 
   // An array of glob patterns indicating a set of files for which coverage information should be collected
-  collectCoverageFrom: [
-    './src/CoinbaseWalletSDK.ts',
-    './src/core/**/*.ts',
-    './src/provider/**/*.ts',
-    './src/relay/**/*.ts',
-    './src/components/**/*.tsx',
-    './src/signer/**/*.ts',
-    './src/transport/**/*.ts',
-  ],
+  collectCoverageFrom: ['./src/**/*.ts'],
   // The directory where Jest should output its coverage files
   coverageDirectory: 'coverage',
 
   // An array of regexp pattern strings used to skip coverage collection
-  coveragePathIgnorePatterns: ['/node_modules/'],
+  coveragePathIgnorePatterns: ['/node_modules/', 'vendor-js/', 'assets/', 'walletlink/relay/'],
```

Comment:
as of now
![image](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/a5a1d71e-be8e-4d9d-850f-a3ea5175e251)